### PR TITLE
Regenerate SDKs with a missing parameter in AddFaceFromStream operation in FaceList

### PR DESCRIPTION
**Pull request by DavidLiCIG**
Regenerate SDKs adding a missing parameter.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
Can you use the latest autorest 2.0.4221 or greater to generate the SDK.

File: src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
```
[the remark this answers is not in the document]
Comment (DavidLiCIG) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/.vscode/launch.json]: Removed

File: src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/.vscode/tasks.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
```
[the remark this answers is not in the document]
Comment (DavidLiCIG) [reply src/SDKs/CognitiveServices/dataPlane/Vision/Vision.Tests/.vscode/tasks.json]: Removed

**Comment by DavidLiCIG** [comment]
@shahabhijeet the latest version of Autorest as shown here is 4215?
https://www.npmjs.com/package/autorest

**Comment by DavidLiCIG** [comment]
Not sure what the error from CI is all about though.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**